tvb_build/hbp_openshift_deployer/tvb_deployer.yml

```diff
@@ -520,5 +520,49 @@ objects:
           targetPort: 5432
       selector:
         deploymentconfig: pg-bouncer
+  - kind: CronJob
```

Comment:
After internal discussions we decided to separate the CronJob in a separate .yml file